x/staking/keeper/delegation.go

```diff
@@ -593,7 +593,7 @@ func (k Keeper) Delegate(
 		}
 
 		coins := sdk.NewCoins(sdk.NewCoin(k.BondDenom(ctx), bondAmt))
-		if err := k.bankKeeper.DelegateCoinsFromAccountToModule(ctx, delegatorAddress, sendName, coins); err != nil {
+		if err := k.bankKeeper.SendCoinsFromAccountToModule(ctx, delegatorAddress, sendName, coins); err != nil {
```

Comment:
`SendCoinsFromAccountToModule` doesn't do extra work(e.g. recording tracking information) regarding the delegation, while `DelegateCoinsFromAccountToModule` does.

Reply:
Yes, this is an intentional change so that the locked amount by vesting cannot be delegated